Description: Add to support disabling rpath in ./configure
  http://icculus.org/pipermail/smpeg/2010-November/000183.html
Author: Michael Smith <msmith at cbnco.com>
Last-Update: 2014-08-09
Applied-Upstream: http://svn.icculus.org/smpeg?view=rev&revision=390

--- trunk/configure.in	2009/10/18 06:08:35	389
+++ trunk/configure.in	2010/12/31 00:16:25	390
@@ -84,19 +84,27 @@
         ;;
 esac
 
-# Set runtime shared library paths as needed 
+dnl Set runtime shared library paths as needed 
+AC_ARG_ENABLE(rpath,
+	AC_HELP_STRING([--enable-rpath],
+		[use an rpath when linking SDL [[default=yes]]]),,
+	enable_rpath=yes)
 
-case "$target" in
-    *-*-linux*)
-	SMPEG_RLD_FLAGS="-Wl,-rpath,\${exec_prefix}/lib"
-	;;
-    *-*-freebsd*)
-	SMPEG_RLD_FLAGS="-Wl,-rpath,\${exec_prefix}/lib"
-	;;
-    *-*-solaris*)
-	SMPEG_RLD_FLAGS="-R\${exec_prefix}/lib"
-	;;
-esac
+if test "x$enable_rpath" = "xyes"; then
+  case "$target" in
+      *-*-linux*)
+	  SMPEG_RLD_FLAGS="-Wl,-rpath,\${exec_prefix}/lib"
+	  ;;
+      *-*-freebsd*)
+	  SMPEG_RLD_FLAGS="-Wl,-rpath,\${exec_prefix}/lib"
+	  ;;
+      *-*-solaris*)
+	  SMPEG_RLD_FLAGS="-R\${exec_prefix}/lib"
+	  ;;
+  esac
+else
+  SMPEG_RLD_FLAGS=""
+fi
 
 AC_SUBST(SMPEG_RLD_FLAGS)
 
